version.go

```diff
@@ -405,3 +406,25 @@ func (v *Version) UnmarshalText(b []byte) error {
 func (v *Version) MarshalText() ([]byte, error) {
 	return []byte(v.String()), nil
 }
+
+// Scan implements the sql.Scanner interface.
+func (v *Version) Scan(src interface{}) error {
+	switch src := src.(type) {
+	case *Version:
+		*v = *src
+		return nil
+	case []byte:
+		return v.UnmarshalText(src)
+	case string:
+		return v.UnmarshalText([]byte(src))
+	case nil:
+		return nil
+	default:
+		return fmt.Errorf("Cannot convert %T to Version", src)
```

Comment:
```suggestion
		return fmt.Errorf("cannot scan %T as Version", src)
```

Reply:
Update the text according to your suggestion 👍🏻